server/seeders/20181026125921-user-admin.js

```diff
@@ -0,0 +1,22 @@
+import passwordUtil from '../utilities/passwordHasher';
+
```

Comment:
Lets call these env variables ADMIN_FIRSTNAME etc

Reply:
Also make sure you add them to heroku

Reply:
Okies...